src/index.ts

```diff
@@ -293,7 +339,11 @@ export default class StatsGatherer extends EventEmitter {
     return true;
   }
 
-  private polyFillStats (results: RTCStatsReport): Array<{key: RTCStatsType, value: any}> {
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  private polyFillStats(
+    results: RTCStatsReport,
+  ): Array<{ key: RTCStatsType; value: any }> {
+    /* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
I don't love how the combination of Prettier and TypeScript-ESLint work in some of these cases, but I'm not sure there's much we can do about it.

Happy to hear ideas, though.